ansible/roles/baselayout/tasks/partials/ccache/macos.yml

```
@@ -8,6 +8,6 @@
 - name: "ccache : add ccache to the path (macos)"
   when: ccache_mac.rc == 1
   lineinfile:
-    dest: "etc/paths"
+    dest: "/etc/paths"
     insertbefore: BOF
     line: "usr/local/opt/ccache/libexec"
```

```suggestion
    line: "/usr/local/opt/ccache/libexec"
```

Sorry, looks like I inadvertently dropped the leading slash in the refactor in https://github.com/nodejs/build/pull/2569/files#diff-01b473334dd8c7ebd76126710bd5ba533d6334a363653425f7d4b27adff36616.